raiden/storage/serialization/types.py

```diff
@@ -0,0 +1,230 @@
+    if bp_type is None:
+        return None
+
+    if bp_type.endswith("UnsignedState"):
+        return SchemaCache.get_or_create_schema(BalanceProofUnsignedState)
+    elif bp_type.endswith("SignedState"):
+        return SchemaCache.get_or_create_schema(BalanceProofSignedState)
+
+    return None
+
+
+def message_event_schema_serialization(message_event: SendMessageEvent, parent: Any) -> Schema:
+    # pylint: disable=unused-argument
+    return SchemaCache.get_or_create_schema(message_event.__class__)
+
+
+def message_event_schema_deserialization(
+    message_event_dict: Dict[str, Any], parent: Dict[str, Any]
+) -> Optional[Schema]:
+    # pylint: disable=unused-argument
+    message_type = message_event_dict.get("_type")
+    if message_type is None:
+        return None
+
+    if message_type.endswith("SendLockExpired"):
+        from raiden.transfer.mediated_transfer.events import SendLockExpired
+
+        return SchemaCache.get_or_create_schema(SendLockExpired)
+    elif message_type.endswith("SendLockedTransfer"):
+        from raiden.transfer.mediated_transfer.events import SendLockedTransfer
+
+        return SchemaCache.get_or_create_schema(SendLockedTransfer)
+    elif message_type.endswith("SendSecretReveal"):
+        from raiden.transfer.mediated_transfer.events import SendSecretReveal
+
+        return SchemaCache.get_or_create_schema(SendSecretReveal)
+    elif message_type.endswith("SendBalanceProof"):
+        from raiden.transfer.mediated_transfer.events import SendBalanceProof
+
+        return SchemaCache.get_or_create_schema(SendBalanceProof)
+    elif message_type.endswith("SendSecretRequest"):
+        from raiden.transfer.mediated_transfer.events import SendSecretRequest
+
+        return SchemaCache.get_or_create_schema(SendSecretRequest)
+    elif message_type.endswith("SendRefundTransfer"):
+        from raiden.transfer.mediated_transfer.events import SendRefundTransfer
+
+        return SchemaCache.get_or_create_schema(SendRefundTransfer)
+
+    elif message_type.endswith("SendProcessed"):
+        from raiden.transfer.events import SendProcessed
+
+        return SchemaCache.get_or_create_schema(SendProcessed)
+
+    return None
+
+
+_native_to_marshmallow.update(
+    {
+        # Addresses
+        Address: AddressField,
+        InitiatorAddress: AddressField,
+        PaymentNetworkID: AddressField,
+        SecretRegistryAddress: AddressField,
+        TargetAddress: AddressField,
+        TokenAddress: AddressField,
+        TokenNetworkAddress: AddressField,
+        TokenNetworkID: AddressField,
+        # Bytes
+        EncodedData: BytesField,
+        AdditionalHash: BytesField,
+        BalanceHash: BytesField,
+        BlockHash: BytesField,
+        Keccak256: BytesField,
+        Locksroot: BytesField,
+        LockHash: BytesField,
+        Secret: BytesField,
+        SecretHash: BytesField,
+        Signature: BytesField,
+        TransactionHash: BytesField,
+        # Ints
```

Comment:
Most of these should be serialized as strings as sqlite ints are only 64bit IIRC.